code/analyticsedge/src/phone/java/com/adobe/marketing/mobile/AnalyticsExtension.java

```diff
@@ -315,10 +334,10 @@ private void track(final Event event) {
         }
 
         // Todo:- Should we append default lifecycle context data (os version, device name, device version, etc) to each hits?
-        final Map<String, Object> contextData = event.getEventData();
+        final Map<String, Object> contextData = (Map<String, Object>) event.getEventData().get(AnalyticsConstants.EventDataKeys.CONTEXT_DATA);
```

Comment:
We can reuse the eventData variable here.

Reply:
The eventData variable above refers to event.GetData(). Here I need Map returns by calling event.getEventData().